docs/src/gs/create-workload-cluster.md

```diff
@@ -11,17 +11,17 @@ further configured  with the parameters below.
-| `OCI_COMPARTMENT_ID`                      |                     | The OCID of the compartment in which to create the required compute, storage and network resources.                                                                                                                              |
-| `OCI_IMAGE_ID`                            |                     | The OCID of the image for the kubernetes nodes. This same image is used for both the control plane and the worker nodes.                                                                                                         |
-| `OCI_CONTROL_PLANE_MACHINE_TYPE`          | VM.Standard.E4.Flex | The [shape](https://docs.oracle.com/en-us/iaas/Content/Compute/References/computeshapes.htm) of the Kubernetes control plane machine.                                                                                            |
-| `OCI_CONTROL_PLANE_MACHINE_TYPE_OCPUS`    | 1                   | The number of OCPUs allocated to the control plane instance.                                                                                                                                                                     |
-| `OCI_NODE_MACHINE_TYPE`                   | VM.Standard.E4.Flex | The [shape](https://docs.oracle.com/en-us/iaas/Content/Compute/References/computeshapes.htm) of the Kubernetes worker machine.                                                                                                   |
-| `OCI_NODE_MACHINE_TYPE_OCPUS`             | 1                   | The number of OCPUs allocated to the worker instance.                                                                                                                                                                            |
-| `OCI_SSH_KEY`                             |                     | The public SSH key to be added to the Kubernetes nodes. It can be used to login to the node and troubleshoot failures.                                                                                                           |
-| `OCI_CONTROL_PLANE_PV_TRANSIT_ENCRYPTION` | true                | Enables [in-flight Transport Layer Security (TLS) 1.2 encryption](https://docs.oracle.com/en-us/iaas/Content/File/Tasks/intransitencryption.htm) of data between control plane nodes and their associated block storage devices. |
-| `OCI_NODE_PV_TRANSIT_ENCRYPTION`          | true                | Enables [in-flight Transport Layer Security (TLS) 1.2 encryption](https://docs.oracle.com/en-us/iaas/Content/File/Tasks/intransitencryption.htm) of data between worker nodes and their associated block storage devices.        |
+| Parameter                                 | Default Value       | Description                                                                                                                                                                                                                                                     |
```

Comment:
shouldnt we add a section here? I am ok as long as we have a new ticket to add docs

Reply:
As of right now I believe this section only contains ENV info for our templates. We discussed not including the region info as part of our templates (yet) there is nothing to add here.

Reply:
I was talking about a section where we explain the region support and how users can customize the template

Reply:
https://github.com/oracle/cluster-api-provider-oci/issues/46